cargo-dist/tests/integration-tests.rs

```diff
@@ -838,6 +838,123 @@ libcue = "2.3.0"
     })
 }
 
+#[test]
+fn axolotlsay_alias() -> Result<(), miette::Report> {
```

Comment:
Yeah, I agree. Can we do that in a followup, though? I'd like reorganizing our tests to be fairly intentional, which means we can think directly about which tests contain which groups of features instead of it being a byproduct of this PR.

Reply:
@mistydemeo a follow up seems good to me- can you file an issue?

Reply:
Done: https://github.com/axodotdev/cargo-dist/issues/979